www/header.php

```diff
@@ -204,3 +242,13 @@
     <div id="cv-playlist">
         <ul class="cv-playlist"></ul>
     </div>
+
+	<!-- Only included when generate index.html for developmed purpose -->
```

Comment:
When generating a development only index.html it should also contain the content out of index.tpl.html and the footer.php.